src/main/java/com/jcabi/aspects/aj/MethodLogger.java

```diff
@@ -240,6 +240,7 @@ private Object wrap(final ProceedingJoinPoint point, final Method method,
                 } else {
                     origin = "somewhere";
                 }
+                level = annotation.exceptionLevel() == -1 ? level : annotation.exceptionLevel();
```

Comment:
Checkstyle will not allow ternary operator

Reply:
solved